client/app/index.js

```diff
@@ -26,6 +34,8 @@ function loadCurrentModule() {
 }
 
 function loadModules() {
+  loadAceEditorDependencies();
```

Comment:
Any reasons that you defined a new method here? should it be merge with `layout_ace_editor.js` ?